zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -443,7 +453,7 @@ pub(crate) fn send_multi_step_proposed_transfer<T: ShieldedPoolTester>() {
 
     assert_eq!(
         confirmed_sent.get(0).and_then(|v| v.get(0)),
-        Some(&(&txids[0], 50000))
+        Some(&(&txids[0], 35000))
```

Comment:
It took me a while to realise that this edit was not due to changes in the existing output amount or how fees were changed, but just checking the value of the new change output instead.

This test now has an undocunmented dependency on implementation details about the order in which the two outputs of the first transaction are added to the database. Change the test to instead sort the outputs by value and check both of them (the existing 50,000 zatoshi non-change output, and the new 35,000 zatoshi change output).